packages/datadog-plugin-aws-sdk/src/index.js

```diff
@@ -0,0 +1,90 @@
+'use strict'
+
+const Tags = require('opentracing').Tags
+const analyticsSampler = require('../../dd-trace/src/analytics_sampler')
+const awsHelpers = require('./helpers')
+
+function createWrapRequest (tracer, config) {
+  config = normalizeConfig(config)
+
+  return function wrapRequest (send) {
+    return function requestWithTrace (cb) {
+      if (!this.service) return send.apply(this, arguments)
+
+      const serviceName = this.service.serviceIdentifier
+      const childOf = tracer.scope().active()
+      const baseTags = {
+        [Tags.SPAN_KIND]: 'client',
+        'span.type': 'http',
+        'service.name': config.service
+          ? `${config.service}-aws-${serviceName}`
+          : `${tracer._service}-aws-${serviceName}`,
+        'aws.operation': this.operation,
+        'aws.region': this.service.config && this.service.config.region,
+        'aws.service': this.service.api && this.service.api.className,
+        'aws.url': this.service.endpoint && this.service.endpoint.href,
```

Comment:
Will this be the same as the `http` plugin `http.url` value? If so, it can probably be omitted.

Reply:
yup it should be, removed